docs/sdk/api/sdkMiddlewareHttp.md

```diff
@@ -21,6 +21,7 @@ Creates a [middleware](/sdk/Glossary.md#middleware) to handle HTTP requests for
 #### Named arguments (options)
 
 1. `host` *(String)*: the host of the HTTP API service
+2. `credentialsMode` *(String)*: one of the supported `credentials` modes (`omit`, `same-origin`, `include`), useful when working with HTTP Cookies. (Default: undefined)
```

Comment:
I don't think the `(Default: undefined)` part is necessary. Makes it seem like `undefined` is a usable mode

Reply:
Yes good point, changed that to `optional` https://github.com/commercetools/nodejs/pull/485/commits/80552b4885b1c4cda35a148e34b06411311fb96a